clang/test/CodeGen/aarch64-elf-pauthabi.c

```diff
@@ -0,0 +1,61 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
Most clang/test/CodeGen tests don't need registered-target. You can check this by removing AArch64 from LLVM_TARGETS_TO_BUILD

Reply:
Ensured that it's not needed and deleted in 2fd8f6606c6d287085582b82145b05a635fbf776, thanks